### PR TITLE
Adds pip support

### LINKED ISSUE #607: pip support
Is there any chance for a pip package that works with populus the way the npm package works with truffle?

I might be up for making a pull request if I could get a little guidance on how it would work.

### PR DESCRIPTION
**Pull request by jessebmiller**
This certainly seems like a temporary solution, and it would be completely reasonable to wait for populus to implement their packaging solution.
Fixes #607

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    for file_name in file_names:
+        file_path = os.path.join(dir_name, file_name)
+        contract_files = data_files.get(dir_name, [])
+        contract_files.append(file_path)
+        data_files[dir_name] = contract_files
+
+setup(
+    name="zeppelin-solidity",
+    version="1.4.0",
+    description="Secure Smart Contract library for Solidity",
```
Comment (federicobond) [inline setup.py]: Please add the OpenZeppelin website URL here.
```
warning: check: missing required meta-data: url 
```

**Comment by federicobond** [comment]
Thank you @jessebmiller! 🙌 Awesome work.

I just tried installing it in virtualenv and contracts end up in `/path/to/virtualenv/contracts/`. Is that expected behaviour? Where does Populus expect to find them? I think we should use a zeppelin directory so they don't get confused with other packages and are clearly recognizable in imports.

**Comment by jessebmiller** [comment]
Hmm no that's not expected, populus as far as I can tell is only looking in `<project root>/contracts`

What pip command did you use? For this to work it's got to specify the project root. So something like `pip install --root /path/to/project/root open-zeppelin` once it's up on PyPI

File: setup.py (hunk 1 above)
Comment (jessebmiller) [reply setup.py]: Sure thing

**Comment by federicobond** [comment]
Hi @jessebmiller, sorry about the delay, I tried installing it from the local copy on my machine, like this:
```
pip install pip install --root . ../zeppelin-solidity
```
But it ends up putting the contracts at:
```
join('/path/to/project/', 'path/to/my/virtualenv/directory/inside/project/', 'contracts/')
```

**Comment by federicobond** [comment]
@jessebmiller can we try absolute paths from where pip was called for `dir_name`? That might work as a workaround for now.

**Comment by jessebmiller** [comment]
I tried to get absolute paths to where pip was called in earlier attempts and I'm not sure that's feasible as pip runs `setup.py` somewhere else. I'll see about looking for the populus config file, but I think I'll run into similar problems.

What happens if you run

`pip install --root $(pwd) ../zeppelin-solidity`?

**Comment by nventuro** [comment]
Closing due to staleness.

**Pull request closed without merging**